core/html_api.php

```diff
@@ -311,6 +311,22 @@ function html_meta_redirect( $p_url, $p_time = null, $p_sanitize = true ) {
 	return true;
 }
 
+/**
+ * Print a canonical meta tag.
+ *
+ * @param string  $p_url      The canonical URL: has to be a relative path.
```

Comment:
Suggest referencing to what the URL must be relative to: 
`The canonical URL: has to be relative to {@see $g_path}`